File: DataBinding/Section/LINQ/VB.NET/LINQ/ViewerForm.vb
Imports System.Xml

Public Class ViewerForm

	Dim _manager As System.Resources.ResourceManager
	'  LINQ to Object
	Private Sub ViewerForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
		_manager = New System.Resources.ResourceManager(Me.GetType)

		' To generate a report.
		Dim rpt As New SectionReport
		rpt = New SectionReport
		rpt.LoadLayout(XmlReader.Create("..\\..\\rptLINQtoObject.rpx"))

		arvMain.LoadDocument(rpt)
	End Sub

End Class

File: DataBinding/Section/LINQ/C#/LINQ/ViewerForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using System.Xml;

namespace GrapeCity.ActiveReports.Samples.LINQ
{
	public partial class ViewerForm : Form
	{
		private System.Resources.ResourceManager _manager;
		public ViewerForm()
		{
			_manager = new System.Resources.ResourceManager(typeof(ViewerForm));
			InitializeComponent();
		}
	  
		// Define a structure for LINQtoObject.
		private void ViewerForm_Load(object sender, EventArgs e)
		{
			// To generate a report.
			var rpt = new SectionReport();
			rpt.LoadLayout(XmlReader.Create("..\\..\\rptLINQtoObject.rpx"));
			// To run the report.
			arvMain.LoadDocument(rpt);
		}
	}
}
